src/xrCore/_math.cpp

```diff
@@ -215,9 +215,49 @@ XRCORE_API u64 QPC() noexcept
     return _dest;
 }
 
+#if defined(XR_ARM)
+#include <sys/syscall.h>
+#include <linux/perf_event.h>
+
+static int fddev = -1;
+__attribute__((constructor)) static void
+init(void)
```

Comment:
It's better to reimplement this via static initializer class.

Reply:
And it's better to move that class upper, closer to the beginning of the file, where you start to include ARM and ARM64 includes.

Reply:
Check last commit.